internal/pkg/task/task.go

```diff
@@ -30,7 +32,13 @@ type DefaultClusterGetter interface {
 
 // TaskRunner runs the tasks and wait for it to start.
 type TaskRunner interface {
-	RunTask(input ecs.RunTaskInput) ([]string, error)
+	RunTask(input ecs.RunTaskInput) ([]*ecs.Task, error)
+}
+
+type Task struct {
+	TaskARN string
+	ClusterARN string
+	StartedAt *time.Time
```

Comment:
nit: Is there a reason why this is a pointer?

Reply:
Also can we run `gofmt` on these files? I think the formatting is off

Reply:
Because I wanted to be able to tell whether `StartedAt` is simply not set (by checking if it's `nil`) or if it's meant to be the zero time.